### DISCUSSION TITLE
External thermostat change

### DISCUSSION THREAD

**Discussion by alexdelli** [raomin/ESPAltherma#182] [Q&A] [answered]
Hi, today I was installing the relais for external thermostat, on X2M 1 and 4 pin there is all the external thermostat for every room.
So, every area with a thermostat, send activation on X2M.
Question is: how can I add the espaltherma thermostat with the priority of other thermostats?

i.e.:  today is sunny, I want to "overheating" the bedroom. Bedroom temp is 19°, thermostat is set to 21°, but I set the espaltherma thermostat to 20°.

I have to disconnect the pin 1 and 4, and connect relais? And how the system knows which pump to use?

thank you

![image](https://user-images.githubusercontent.com/6674409/199043773-5da66173-4afe-4077-9e71-a8151643cd93.png)

**Comment by mauromorello** [comment, marked as answer]
Hi Mauro here. I'm from Italy but I try to write in english in order to help someone else is trying to understand this topic. I'm in a strange situation, where I found my Altherma with X2M->1 already connected (it should be the thermostat ON/OFF), but when I try to manually change the state of the thermostat (setting I.E an higher setpoint) i cannot see any changes on thermostat value in ESPAltherma data. I'm starting to think my heater is not well configured. So it runs continuously (despite of thermostat), the way the system works it beacuse the thermostat control also the water circuit, "cutting" the long loop when the temperature is reached. Do you know about panel settings?

**Reply by Tardymo** [reply]
What model is your heat pump and which thermostat do you have?

**Reply by mauromorello** [reply]
My pump model is EHVX08626CB9W and thermostat is by Bticino; but between thermostat and pump there is the control of solenoid valve used to open or close water circulation on floor. I understood that when thermostat is "on" it switch to "open" to water circuit, and in the meantime (by a relè) it should be switch on also Altherma thermostat input. By now I bypassed all of there by putting a "Wifi-Finger" (AKA switchbot bot) controlled by home assistant, that phisically turn ON and OFF all daikin altherma. By doing this my electrical consumption drop from 800kWh (JAN 2022) to 500kWh (JAN 2023) having almost the same temperature at home.

**Comment by Tardymo** [comment]
Were you able to solve the problem with you thermostats?
External thermostat just tells you heat pump to turn on or off. You can have more than one in parallel. If at least one room needs more heat, your heat pump will turn on.
You can also connect esparltherma relay in parallel to your existing thermostats, but there is no easy way to prioritize it over an existing thermostat in the same room. And I can't see a reason why you would want two thermostats in one room. If you want to control your room using espaltherma, then disconnect the existing thermostat.

**Comment by alexdelli** [comment]
Yes, I solved using Shelly 1 for every thermostat. In this way I have the control of each circulation pump for every zone, the control for each room, the control of heat pump and I kept the thermostats function to manual on/off for every emergency needs

**Comment by Edsol** [comment]
I am resurrecting this old post (I am also Italian 😄 ).

I have a similar problem and can't figure out where the problem is. I came to understand from the manual that the PINs for turning the heating on and off via ESPAltherma are X2M 1-4 (as you suggested). I have configured a clean contact relay and via ESPAltherma it works correctly as I can hear the relay clicking but nothing happens on the heater.

Any suggestions as to where the problem might be? I get the doubt that the relay must not be a dry contact but powered? although this sounds strange to me frankly.

**Comment by mauromorello** [comment]
Hi (Italian here) if your heater is the same of mine, you have to set
"External Thermostat" into heater service menu.
Is well explained in downloadable manuals
Notice that also the value of the thermostat (ON and OFF) are inside the
parameters:

RT setpoint: 21 Main RT Cooling: OFF Main RT Heating: ON

RT setpoint is not operable, but you see the effects of relay open / close

Il giorno lun 13 gen 2025 alle ore 15:43 Edsol ***@***.***>
ha scritto:

**Comment by Edsol** [comment]
My heater model is EHYKOMB33AA + EHYHBX08, I have already made an attempt to enable the A.2.1.7 or A.2.2.4 voice menu but nothing changes.
